Makefile

```diff
@@ -2,9 +2,15 @@
 django-tests:
 	python manage.py test
 
-performance-tests:
+load-tests:
```

Comment:
nit-picky, but I've heard "load tests" imply tests on a system to see how much load it can take before it breaks. While this test is more like a performance regression test?